sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_version.py

```diff
@@ -4,4 +4,4 @@
 # license information.
 # -------------------------------------------------------------------------
 
-VERSION = "1.2.0b1"
+VERSION = "1.3.0.dev1"
```

Comment:
is this a new version format? Why not "1.3.0.b1"?

Reply:
discussed offline. this is introduced to track dev usage of the source code, as opposed to official package usage.